src/language.h

```diff
@@ -0,0 +1,19 @@
+#ifndef LANGUAGE_H
+#define LANGUAGE_H
+
+#include <QString>
+#include <QMap>
+
+class Language
```

Comment:
This class is more about translation or text provider. Language seems a bit to generic.